package.json

```diff
@@ -64,6 +64,7 @@
     "ts-node": "~3.2.0",
     "tslint": "~5.3.2",
     "tslint-eslint-rules": "^4.1.1",
-    "typescript": "2.4.2"
+    "typescript": "2.4.2",
+    "webpack": "^3.9.1"
```

Comment:
why webpack is added?

Reply:
it says to update webpack/lib/node/NodeTemplatePlugin thats why wepack is added while updating NodeTemplatePlugin

Reply:
yes checked with my machine without webpack and it works. So it doesn't need to add.